api/valor_api/schemas/evaluation.py

```diff
@@ -167,6 +169,7 @@ class EvaluationResponse(BaseModel):
     confusion_matrices: list[ConfusionMatrixResponse] | None = None
     ignored_pred_labels: list[Label] | None = None
     missing_pred_labels: list[Label] | None = None
+    created_at: datetime.datetime
```

Comment:
- can you update the docstring to include `created_at`?
- nit: should `created_at` be moved above the optional arguments for consistency (e.g., moving it above `metrics`)?

Reply:
yes good point, just made those changes